apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -167,42 +167,55 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
                     </IconButton>
                 </Tooltip>
             )}
-            <Dialog open={open} onClose={handleClose} maxWidth={'lg'}>
+            <Dialog open={open} onClose={handleClose} maxWidth={'xs'}>
+                <DialogTitle id="form-dialog-title">
+                    {props.customEvent ? 'Edit a Custom Event' : 'Add a Custom Event'}
+                </DialogTitle>
                 <DialogContent>
```

Comment:
```suggestion
                <DialogContent sx={{ display: 'flex', flexDirection: 'column', gap: 1 }}>
```

Similar to how we got rid of `marginLeft` and `marginRight` we should also avoid using margin across our components in favor of using the `gap` styling at the parent container